apogee/tools/read.py

```diff
@@ -1265,8 +1281,13 @@ def _swap_in_astroNN(data,astroNNdata):
     return data
 
 def _add_astroNN_distances(data,astroNNDistancesdata):
+    dr= path._default_dr()
     fields_to_append= ['dist','dist_model_error','dist_error',
                        'weighted_dist','weighted_dist_error']
+    if int(dr) == 16:
+        #also have galactocentric and orbit info
+        fields_to_append= ['dist','dist_model_error','dist_error',
+                           'weighted_dist','weighted_dist_error']
```

Comment:
Why did you add this? I think this code also works for DR14...

Reply:
Sorry, this got left in from a previous version I was using to test things - removed in the new version